### DISCUSSION TITLE
Improved db2 profile invalid user credentials management

### DISCUSSION THREAD

**Discussion by Rune-Christensen** [zowe/zowe-cli-db2-plugin#131] [Q&A]
I am unsure if this request belongs here, or in the VS Code Zowe Explorer project.

As a company that wants to roll out more usage of Zowe amongst our developers, especially using VS Code, 
We are running into issues with user accounts getting suspended on our Z, because the Zowe DB2 profile does not get updated, together with the regular Zowe profile, when Zowe detects that the password is incorrect, when using VS Code. 
This leads to frustration amongst our developers, as it is not obvious what causes their accounts to get suspended.

Please implement, that requests to Z from Zowe DB2, does not continue after the first invalid password response is received from a call to Z. 
If possible, also implement a way to easily update the password in VS Code, preferably at the same time as the update of the regular Zowe profile.

**Comment by zFernand0** [comment]
Hey @Rune-Christensen,
Would it help to share those credentials with a `base` profile or restructuring your team-config file to allow shared credentials through "profile nesting"?
